contracts/Voting.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract Voting {
+    struct Poll {
+        uint id;
+        string title;
+        string description;
+        address creator;
+        mapping(uint => Option) options;
+        uint optionCount;
+        mapping(address => bool) hasVoted;
+    }
+
+    struct Option {
+        uint id;
+        string text;
+        uint voteCount;
+    }
+
+    mapping(uint => Poll) public polls;
+    uint public pollCount;
+
+    modifier onlyCreator(uint _pollId) {
+        require(
+            msg.sender == polls[_pollId].creator,
+            "Only the poll creator can add options"
+        );
+        _;
+    }
+
+    modifier onlyOnceVote(uint _pollId) {
+        require(
+            !polls[_pollId].hasVoted[msg.sender],
+            "You have already voted in this poll"
+        );
+        _;
+    }
+
+    event CreatePoll(
+        address indexed creator,
+        string _title,
+        string _description
+    );
+    event AddOption(address indexed creator, uint _pollId, string _text);
```

Comment:
CreatePoll and AddOption looks like a function name.
Rename them to be looked as events. For example, PollCreated or Poll Added